packages/sdk/src/HopBridge.ts

```diff
@@ -801,7 +774,47 @@ class HopBridge extends Base {
       this.getTotalDebit(destinationChain, bonder)
     ])
 
-    const availableLiquidity = credit.sub(debit)
+    let availableLiquidity = credit.sub(debit)
+
+    const isBondableChain = bondableChains.includes(sourceChain.slug)
+    const includePendingAmount =
+      isBondableChain && destinationChain.equals(Chain.Ethereum)
+
+    if (includePendingAmount) {
+      const bridgeContract = await this.getBridgeContract(sourceChain)
+      const pendingAmount = await bridgeContract.pendingAmountForChainId(
```

Comment:
IIUC this can be removed.

Reply:
remove the pending amounts subtraction?

Reply:
my understanding was having `((availableLiquidity - pendingAmounts) / 2) - 50k`

Reply:
Maybe I'm not seeing it correctly, but it looks like this line 785 is not doing anything?

The equation you said is on line 812.

Reply:
sorry i see the duplication now, will remove